cloudberry/neo/conf/application.conf

```diff
@@ -89,6 +89,7 @@ berry.query.gap = "1 day"
 #asterixdb.url = "http://localhost:19002/aql"
 #asterixdb.url = "http://uranium.ics.uci.edu:19002/query/service"
 asterixdb.url = "http://localhost:19002/query/service"
+#asterixdb.url = "http://128.195.52.124:19002/query/service"
```

Comment:
Let's not commit changes to `application.conf` configuration file since it's irrelevant to this PR.